{{ cookiecutter.package_name }}/.github/workflows/ci.yml

```
@@ -35,15 +35,15 @@ jobs:
       coverage: codecov
       toxdeps: tox-pypi-filter
       envs: |
-        - linux: py313
+        - linux: py314
```

too soon, because of lacking wheels
```suggestion
        - linux: py313
```